src/DotNetWorker.ApplicationInsights/Initializers/TokenCredentialOptions.cs

```diff
@@ -42,64 +42,64 @@ public static TokenCredentialOptions ParseAuthenticationString(string applicatio
 
             var tokenCredentialOptions = new TokenCredentialOptions();
             bool isValidConfiguration = false;
+
             foreach ((int, int) split in Tokenize(applicationInsightsAuthenticationString))
```

Comment:
There's still an opportunity to improve the tokenization, but opted not to do it for now as the ideal API (Split) is not available in .NET standard and the typical use case here will yield 2 ranges.